tests/tests.hh

```diff
@@ -25,6 +25,8 @@ __cheri_compartment("debug_test") int test_debug_cxx();
 __cheri_compartment("debug_test") int test_debug_c();
 __cheri_compartment("unwind_cleanup_test") int test_unwind_cleanup();
 
+__cheri_compartment("test_runner") int print_version_information();
```

Comment:
```suggestion
int print_version_information();
```

I don't think the compartment annotation is necessary here, unless you intend to export it as a compartment entry point?

Reply:
Oh, indeed, just like `test_global_constructors()` below it.  Good catch; thanks!